src/funfuzz/ccoverage/get_build.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Downloads coverage builds and other coverage utilities, such as grcov.
+"""
+
+from __future__ import absolute_import, division, unicode_literals  # isort:skip
+
+import io
+import logging
+import os
+import tarfile
+import zipfile
+
+import requests
+
+RUN_COV_LOG = logging.getLogger("funfuzz")
+
+
+def get_coverage_build(dirpath, args):
+    """Gets a coverage build from a specified server.
+
+    Args:
+        dirpath (Path): Directory in which build is to be downloaded in.
+        args (class): Command line arguments.
+    """
+    RUN_COV_LOG.info("Downloading coverage build zip file into %s from %s", str(dirpath), args.url)
+    with requests.get(args.url, stream=True) as f:
+        build_request_data = io.BytesIO(f.content)
+
+    RUN_COV_LOG.info("Extracting coverage build zip file...")
+    build_zip = zipfile.ZipFile(build_request_data)
+    extract_folder = dirpath / "cov-build"
+    extract_folder.mkdir(parents=True, exist_ok=True)  # Ensure this dir has been created for Python 3.5 reasons
```

Comment:
Can you say more about "Python 3.5 reasons"?

Reply:
Yes! `.extractall` in the following line does not automatically create intermediate folders that do not exist, in Python 3.5, but seems to do so in 3.6+. I'll add this in an additional comment.